src/server/index.jsx

```diff
@@ -142,12 +145,12 @@ const injectDefaultCacheHeader = (req, res, next) => {
   const defaultMaxAge = getDefaultMaxAge(req);
   const maxAge =
     req.originalUrl.indexOf('/topics/') !== -1
-      ? defaultMaxAge * 2
+      ? defaultMaxAge * 4
       : defaultMaxAge;
   res.set(
     'Cache-Control',
     `public, stale-if-error=${
-      maxAge * 4
+      maxAge * 10
```

Comment:
Isn't this meaningless: `stale-while-revalidate=${maxAge}, max-age=${maxAge}` ? A `stale-while-revalidate` and `max-age` being the same doesn't allow a window to serve stale while repopulating the cache in the background. As above, for topics it's a no brainer this should be 10 minutes for `stale-while-revalidate`. We should probably apply a lesser value for other assets as well?